src/sage/rings/ring.pyx

```diff
@@ -70,12 +70,12 @@ This is to test a deprecation::
     sage: from sage.rings.ring import CommutativeAlgebra
     sage: class Nein(CommutativeAlgebra):
     ....:     pass
-    sage: F = Nein(QQ, QQ)
+    sage: F = Nein(QQ)
```

Comment:
why this change?

Reply:
to enforce similarity with the Algebra case ; the only mandatory argument is the basering